auction-server/src/auction/repository/submit_auction.rs

```diff
@@ -14,12 +14,15 @@ use {
 };
 
 impl<T: ChainTrait> Repository<T> {
-    #[tracing::instrument(skip_all)]
+    #[tracing::instrument(skip_all, name = "submit_auction_repo", fields(auction_id, tx_hash))]
     pub async fn submit_auction(
         &self,
         auction: entities::Auction<T>,
         transaction_hash: entities::TxHash<T>,
     ) -> anyhow::Result<entities::Auction<T>> {
+        tracing::Span::current().record("auction_id", auction.id.to_string());
+        tracing::Span::current().record("tx_hash", format!("{:?}", transaction_hash));
```

Comment:
is this a reasonable format? From what I remember, it outputs an array of u8 which is hard to parse

Reply:
Maybe this issue was with permission_key?